### PR TITLE
Feature/add balrog

### PR DESCRIPTION
**Pull request by johnmcc**

### CODE REVIEW

**Review by ajrussellaudio — APPROVED** [review]

File: src/fighters/Balrog.rb [hunk 1]
```diff
@@ -4,7 +4,6 @@
 class Balrog < Fighter 
   def initialize ai
     super ai
-
```
Comment (ajrussellaudio) [inline src/fighters/Balrog.rb]: Hiya.

File: src/fighters/Balrog.rb [hunk 2]
```diff
@@ -4,6 +4,7 @@
 class Balrog < Fighter 
   def initialize ai
     super ai
+
```
Comment (ajrussellaudio) [inline src/fighters/Balrog.rb]: Most action my GitHub has seen in weeks. We're all Bitbucket and JIRA.

I miss GitHub :cry:

Comment (johnmcc) [reply src/fighters/Balrog.rb]: Cannae decide if I want that line break in or not.

Comment (johnmcc) [reply src/fighters/Balrog.rb]: Trying to set up Gitlab to run tests automatically. Webhooks aren't doing what I want them to do. Yet.

**Pull request closed without merging**